types/react-native/index.d.ts

```diff
@@ -8533,8 +8533,21 @@ export namespace Animated {
     type EndCallback = (result: EndResult) => void;
 
     export interface CompositeAnimation {
+        /**
+         * Animations are started by calling start() on your animation.
+         * start() takes a completion callback that will be called when the
+         * animation is done or when the animation is done because stop() was
+         * called on it before it could finish.
+         */
```

Comment:
Can you add the doc for the `callback` parameter as well?

Reply:
Done! I also copied the example over :)